test/packaging/ansible/roles/uninstall/files-Linux.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Is this file for `uninstall` or `clear state` after given test?

Reply:
I think, it can be used for both, don't you?

Reply:
I would split `uninstall / purge agent from vm for next tests` - in 2 diff roles. But this is my opinion.

Reply:
done